justfile

```diff
@@ -8,9 +8,40 @@ HOME_DIR := env_var_or_default('HOST_HOME', env_var('HOME'))
 
 HOSTNAME := env_var_or_default('HOSTNAME', '')
 
+# are we attached to a terminal?
+interactive := `tty -s && echo "-it" || echo ""`
+
 # Runs Go unit tests
-test args='':
-    go test ./... {{args}}
+test *args:
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    test_args="{{ args }}"
+    test_args=${test_args:-./...}
+    go test -short ${test_args[*]}
+
+# Runs Go unit tests with docker-compose
+test-integration *args:
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # When run in GitHub Actions, the following is set to 1
+    export AD_CI=${AD_CI:-0}
```

Comment:
Is this used anywhere?

Reply:
Actually, no. It seemed potentially useful, and I didn't prioritize removing it, but I should probably go ahead and do so. I'll do that in a new PR to avoid a bunch of rebasing.